MetricsReloaded/metrics/pairwise_measures.py

```
@@ -81,9 +91,26 @@ def __init__(self, pred, ref, list_values, measures=[], dict_args={}):
+
         }
 
     def expected_cost(self):
+        """
+        Calculates the expected cost defined as:
+
+        Luciana Ferrer - Analysis and comparison of classification metrics - https://arxiv.org/pdf/2209.05355
+
+        .. math::
+
+            EC = \sum_{r}\sum_p c_{rp} P_rD_{rp}
```

Ruff (0.15.9)

[warning] Invalid escape sequence: `\s`

Use a raw string literal

(W605)